src/Braille/BrailleStream.tsx

```diff
@@ -102,33 +105,33 @@ class BrailleStream extends LocalStorageComponent<Props, State, SavedState> {
     let handled = false;
 
     if (ev.keyCode === 8) { // Backspace
-      this.handleBackspace();
+      this.onBackspaceClick();
       handled = true;
     } else if (ev.keyCode === 13 || ev.charCode === 32) { // Enter or Space
-      this.handleNext();
+      this.onNextClick();
       handled = true;
     } else if (ev.charCode >= 49 && ev.charCode <= 54) { // '1' through '6'
-      this.handleClick(Math.pow(2, ev.charCode - 49));
+      this.onCharacterClick(Math.pow(2, ev.charCode - 49));
```

Comment:
Could this be wrapped in a helper function "brailleDotFromIndex" or similar?  I eventually realized what it was doing (probably would have been faster post-coffee), but a comment or a descriptive function wrapper could help future maintenance.